web-app/README.adoc

```diff
@@ -90,6 +90,11 @@ Code for the app. Refer to https://angular.io/guide/styleguide#application-struc
 === e2e
 Location for all e2e tests.
 
-=== Errors
-Due to node_modules not being kept track by git, if there are errors in running the web-app side, run the command 'npm install'. This is due to some missing files on the node_modules side.
+== Install Dependencies
+Due to certain dependencies having vulnerabilities or possibly being malicious, please run these commands so the right version of the dependecy is used.
```

Comment:
They should not need to manually install each dependency. 

Every time a new package is installed, the `package-lock.json` file is updated. If we commit that file, then the next time the person just has to run `npm install` in that folder to get the right dependencies.